<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlDialog</class>
 <widget class="QDialog" name="SqlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>连接名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>登录名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
     <item>
      <widget class="QLabel" name="pwdlabel">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwdEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
     <item>
      <widget class="QLabel" name="databaseLabel">
       <property name="text">
        <string>数据库名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="databaseEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connButton">
       <property name="text">
        <string>保   存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
